.expeditor/verify.pipeline.yml

```diff
@@ -8,6 +8,15 @@ steps:
     executor:
       docker:
         image: ruby:2.6-stretch
+- label: verification-unit-tests-ruby-2.6
```

Comment:
We should just call this component-tests. They're not unit tests really for a lot of things.